src/clients/go/tb_client_test.go

```diff
@@ -14,10 +15,11 @@ import (
 )
 
 const (
-	TIGERBEETLE_PORT                 = "3000"
-	TIGERBEETLE_CLUSTER_ID    uint64 = 0
-	TIGERBEETLE_REPLICA_ID    uint32 = 0
-	TIGERBEETLE_REPLICA_COUNT uint32 = 1
+	TIGERBEETLE_PORT                   = "3000"
+	TIGERBEETLE_CLUSTER_ID      uint64 = 0
+	TIGERBEETLE_REPLICA_ID      uint32 = 0
+	TIGERBEETLE_REPLICA_COUNT   uint32 = 1
+	TIGERBEETLE_CONCURRENCY_MAX uint   = 4096
```

Comment:
Should we bump this to 8192 to reflect the value in `tb_client/context.zig`?

Reply:
Looks like the way we handle `TooManyOutstanding` isn't quite right. This test exposes a bug when running with `concurrency_max` higher than the size of the `DemuxPool` (which is 4096).

Reply:
Ah I see. tb_client wasn't adjusted properly after DemuxPool was introduced. It should be addressed by the intrusive packets PR at least.